src/brouwers/kitreviews/templates/kitreviews/find_kit.html

```diff
@@ -15,50 +15,74 @@
     </form>
 
     {% if form.data %}
-    <h3>{% blocktrans with num_results=kits.count %}Search results ({{ num_results }}){% endblocktrans %}</h3>
+        <h3>{% blocktrans with num_results=kits.count %}Search results ({{ num_results }}){% endblocktrans %}</h3>
 
-    <section class="kit__list">
-        <div class="row">
-        {% for kit in kits %}
-            <div class="col-xs-6 col-sm-4 col-md-3 col-xl-2">
-                <article class="kit__preview" data-id="{{ kit.id }}">
+        {% if not kits.exists %}
+
+            <h4>
+                {% blocktrans %}Oops, we don't have this kit in our database! Would you like to add
+                    it?{% endblocktrans %}
+            </h4>
+
+            <div class="center">
+                <input type="button" data-target="#add-kit-modal" value="{% trans 'Add kit' %}"
+                       class="btn bg-main-orange"/>
+            </div>
+
+            {% add_kit_modal form=add_form data=form.cleaned_data %}
+
+        {% endif %}
+
+        <section class="kit__list">
+            <div class="row">
+                {% for kit in kits %}
+                    <div class="col-xs-6 col-sm-4 col-md-3 col-xl-2">
+                        <article class="kit__preview" data-id="{{ kit.id }}">
                     <span class="h5 kit__preview__name" title="{{ kit.name }}">
                         <strong>{{ kit.name }}</strong>
                     </span>
-                    {% if kit.has_box_image %}
-                        {% thumbnail kit.box_image "300x225" crop="center" as im %}
-                            <img src="{{ im.url }}" class="img-responsive">
-                        {% endthumbnail %}
-                    {% else %}
-                        <img src="{% static "images/thumb.png" %}" class="img-responsive">
-                    {% endif %}
-
-                    <span class="kit__preview__details">
-                        {{ kit.brand }}, {{ kit.scale }}{% if kit.category %}, {{ kit.category }}{% endif %}{% if kit.kit_number %}, <em>{{ kit.kit_number }}</em>{% endif %}
+                            {% if kit.has_box_image %}
+                                {% thumbnail kit.box_image "300x225" crop="center" as im %}
+                                    <img src="{{ im.url }}" class="img-responsive">
+                                {% endthumbnail %}
+                            {% else %}
+                                <img src="{% static "images/thumb.png" %}" class="img-responsive">
+                            {% endif %}
+
+                            <span class="kit__preview__details">
+                        {{ kit.brand }}, {{ kit.scale }}{% if kit.category %}, {{ kit.category }}{% endif %}
+                                {% if kit.kit_number %}, <em>{{ kit.kit_number }}</em>{% endif %}
```

Comment:
don't break this newline, you'll now end up with something like `Revell, 1/72, airplanes , <em>123456</em>` (space before the comma)